### PR TITLE
[WINDUPRULE-431 add rule for the components that was separated out of camel-core

### PR DESCRIPTION
**Pull request by mmelko**
JIRA issue:  https://issues.redhat.com/browse/WINDUPRULE-431

### CODE REVIEW

**Comment by windupgithubbot1** [comment]
Can one of the admins verify this patch?

**Comment by mmelko** [comment]
I'd like to add also check for xstl components in java files. What is the preffered way? Create additional condition in current rule or create new let say `java-moved-component` rule

**Comment by mrizzi** [comment]
ok to test

**Comment by mmelko** [comment]
@mrizzi any feedback?

File: rules-reviewed/camel3/camel2/tests/data/xml-moved-components/pom.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	<artifactId>sample-project</artifactId>
+	<version>0.0.1</version>
+	<name>Sample Project</name>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.apache.camel</groupId>
+			<artifactId>camel-core</artifactId>
+		</dependency>
+	</dependencies>
```
Comment (mrizzi) [inline rules-reviewed/camel3/camel2/tests/data/xml-moved-components/pom.xml]:
```suggestion
		<dependency>
			<groupId>org.apache.camel</groupId>
			<artifactId>camel-seda</artifactId>
		</dependency>
	</dependencies>
```

**Comment by mmelko** [comment]
@mrizzi I believe you mean for some of the components and not for all, right ?

**Comment by mrizzi** [comment]
@mmelko yep sure :wink: the one i proposed and another one would be fine.

**Comment by mmelko** [comment]
@mrizzi thanks. Done

**Comment by mrizzi** [comment]
retest

**Pull request merged**